csm/plugins/cortx/alert.py

```diff
@@ -298,9 +298,20 @@ def _convert_to_csm_schema(self, message):
                 from SSPL's info section of the alert message.
                 5. This string uniquely identifies the resource for which an
                 alert has come.
+                6. Enclosure alerts can be generated from any node. So one alert
+                is generated from one node and another alert for the same enclosure
+                componenet is generated from another node, the status should get
+                updated accordingly. That is why node_id is ignored in case of
+                enclosure while creating sensor_info.
                 """
-                csm_schema[const.ALERT_SENSOR_INFO] = \
-                    '_'.join(str(x) for x in csm_schema[const.ALERT_SENSOR_INFO].values())
+                if res_split[0] == const.ENCLOSURE:
```

Comment:
@kupranay Is this required for R2 since we will only have 1:1 mapping for node server and storage_encl?

Reply:
Good point.. this is not required for R2. @soniyamoholkar  please remove it

Reply:
@kupranay and @pawankumarsrivastava  Removed the check.